src/client/Microsoft.Identity.Client/Platforms/Features/WamBroker/WamBroker.cs

```diff
@@ -556,14 +629,27 @@ private IntPtr GetParentWindow(CoreUIParent uiParent)
             return provider;
         }
 
-
-
         public async Task<MsalTokenResponse> AcquireTokenSilentDefaultUserAsync(
             AuthenticationRequestParameters authenticationRequestParameters,
             AcquireTokenSilentParameters acquireTokenSilentParameters)
         {
             using (_logger.LogMethodDuration())
             {
+                var defaultAccountProvider = await _webAccountProviderFactory.GetDefaultProviderAsync().ConfigureAwait(false);
+                if (defaultAccountProvider == null)
+                {
+                    throw new MsalUiRequiredException(
+                        MsalError.InteractionRequired,
+                        "A default account was not found");
+
+                }
+                // special case: passthrough + default MSA account. Need to use the transfer token protocol.
```

Comment:
Wonder if it's worth it to document all these cases, maybe in OneNote, as a matrix of which accounts (AAD/MSA, MSA-PT/ default MSA/AAD) can use which authorities/tenants via WAM/browser. Seems like quiet a few scenario variations.